src/flash-source-buffer.js

```diff
@@ -2,7 +2,7 @@
  * @file flash-source-buffer.js
  */
 import videojs from 'video.js';
-import muxjs from 'mux.js';
+import flv from 'mux.js/lib/flv';
```

Comment:
What about making `mux.js` have a top-level `flv` file that we can use instead. So, that we do `import flv from 'mux.js/flv'`. I think it'll be a bit cleaner and signal that it more explicitly inside of muxjs that we shouldn't be touching that file willy nilly. As is, importing directly into `lib` feels dirty for me.